src/PublicShareAuthSidebar.vue

```diff
@@ -204,16 +204,24 @@ export default {
 }
 </script>
 
+<style lang="css">
+#talk-sidebar,
+#talk-sidebar *,
+#talk-sidebar *::before,
+#talk-sidebar *::after {
+	box-sizing: border-box;
+}
+</style>
+
 <style lang="scss" scoped>
 @import './assets/variables';
 
-/* Properties based on the app-sidebar */
+/* Styles based on the NcAppSidebar */
 #talk-sidebar {
 	position: relative;
 	flex-shrink: 0;
-	width: 27vw;
-	min-width: 300px;
-	max-width: 500px;
+	width: clamp(300px, 27vw, 500px);
+	height: 100%;
```

Comment:
No parent has a determined height. You can set` height: 100%` in `body.talk-sidebar-enabled` or use `vh` to not be dependent.

Reply:
Adjusted in external css, please give it a try again.
I've tested it in FF in Chrome